knowledge-base/grid-filterrow-bool-customization.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Change the Boolean Filtering Options in a FilterRow Dropdown List
+description: Learn how to customize the default text in a Grid FilterRow dropdown list, including changing "is true" and "is false" to "Yes" and "No," modifying the FilterRow dropdown list display names, and filtering the boolean column values by null for a more user-friendly and tailored filtering experience.
+type: how-to
+page_title: How to Change the Boolean Filtering Options in a FilterRow Dropdown List
+slug: grid-filterrow-bool-customization
+ticketid: 1575619, 1603347, 1646267, 1654640
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Grid for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+This KB article answers the following questions:
+
+* How to change default text `is true` and `is false` in `Grid FilterRow` dropdown list?
+* How to customize the `FilterRow` values?
+* How to change the default display names in `FilterRow` dropdown list?
+* How to filter a boolean `Grid` column values by `null`?
+* How to change the default values in FilterRow dropdown list to "**Yes**" and "**No**"?
+
+## Solution
+
+* Define a [Filter Row Template]({%slug grid-templates-filter%}#filter-row-template).
+* Create a `DropDownList` that includes the custom display values.
+* Create a `Button` that will replicate the `FilterRow` clear button.
```

Comment:
We have a separate related KB article about this, which you can link: https://docs.telerik.com/blazor-ui/knowledge-base/grid-filter-operator-dropdown